modules/nf-core/gsea/gsea/meta.yml

```diff
@@ -0,0 +1,143 @@
+name: "gsea_gsea"
+description: run the Broad Gene Set Enrichment tool in GSEA mode
+keywords:
+  - gene set analysis
+  - enrichment
+  - gsea
+  - gene set
+tools:
+  - "gsea":
+      description: "Gene Set Enrichment Analysis (GSEA)"
+      homepage: "http://www.gsea-msigdb.org/gsea/index.jsp"
+      documentation: "https://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Main_Page"
+      doi: "10.1073/pnas.0506580102"
+      licence: "['BSD-3-clause']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing metadata, at a minimum an id and the reference
+        and target levels e.g. e.g. [ id:'test', reference:"WT", target:"MUT" ].
+        Reference and target levels can then be provided to the module via an ext.args
+        specification for the process like: ext.args = { [ "reference":
+        "$meta.reference", "target": "$meta.target" ] }
```

Comment:
If we go with this method, then should specif that `reference` and `target` keys are mandatory